docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Reviews Rating
+---
+
+The following documentation goes through an example implementation of a review rating platform for the food service industry on Sui. By leveraging blockchain to allow reviewers to own and monetize their feedback, the on-chain platform ensures transparency and temper-proof ratings.
+
+## Personas
+
+There are four actors in the typical workflow of the reviews rating example.  
+
+```mermaid
+sequenceDiagram
+  Service ->> Dashboard: Add service to dashboard.
+  Service ->> User: Send proof of experience.
+  User ->> Service: Send review
+  Service ->> User: Send reward
+  Service ->> Moderator: Add as a service moderator
+  Moderator ->> Service: Remove abused review
+```
+
+### Service owners
+
+Service owners are entities like restaurants that list their services on the platform. Service owners allocate a specific amount of SUI as a reward pool. Assets from the pool are used to provide rewards for honest reviews. A Proof of Experience (PoE) NFT confirms the reviewer used the service, which they burn to provide a verified review. Service owners can provide their customers with unique identifiers, perhaps using QR codes, to identify individual reviewers.
+
+### Reviewers
+
+Reviewers are consumers of services that use the review system. Reviewers provide feedback in the form of comments that detail specific aspects of the service as well as a star rating to inform others. The reviews are rated, with the most effective reviews getting the highest rating. Service owners award the 10 highest rated reviews for their service. 
+
+### Review readers
+
+Review readers access reviews to make informed decisions on selecting services. Readers rate reviews by casting up or down votes. The review readers' ratings are factored into the algorithm that rates the reviews, with the authors of the highest-rated reviews getting awarded. 
+
+### Moderator
+
+The moderator monitors content of the reviews and can delete any reviews that contain inappropriate content.
+
+## How reviews are rated
+
+The reviews are rated (scored) on-chain using following criteria
+- Intrinsic score (IS)
+  - Length of content
+- Extrinsic score (ES)
+  - Number of votes received
+- Verification multiplier (VM)
+  - Reviews with PoE get a multiplier to improve rating
+
+```
+Total Score = (IS + ES) * VM
+```
+
+## Backend
```

Comment:
@amnn can provide some guidance, I'm sure, but we could use some discussion on any common coding patterns used in these modules. Include some reasoning behind why you're choosing to use those patterns would help, too.

Reply:
For this PoC, there is no need to run a backend server. Every operation that is typically done on the backend (ie. sorting reviews) is done on-chain: scoring a review and sorting reviews such that a high-rated review comes first. The reviews are always sorted on-chain and the frontend can just display reviews.

Reply:
Well, we've been calling these contracts the "backend" for these examples, even though Sui is technically the service. Maybe should rename.

Reply:
I've renamed the header "Backend" to "Smart contracts". I see that Blackjack poc also has the header "Smart contracts" [https://docs.sui.io/guides/developer/app-examples/blackjack#smart-contracts]